### DISCUSSION TITLE
How to set defaultProps on components in theme?

### DISCUSSION THREAD

**Discussion by telmen** [mantinedev/mantine#2559] [Q&A]
Setting default `errorProps` on any component in theme customization, results in the following error appearing in the console.

```js
{
...
components: {
  Radio: {
    defaultProps: {
      errorProps: { somePropsHere }  
    }
  }
}
...
}
```

React does not recognize the `errorProps` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `errorprops` instead. If you accidentally passed it from a parent component, remove it from the DOM element.

**Comment by rtivital** [comment]
Because Radio component does not support `errorProps` prop, you can find supported props under `Component Props` tab in documentation.

**Reply by telmen** [reply]
Thanks!
